spec.md

```diff
@@ -1,4 +1,4 @@
-# Open Service Broker API
+# Open Service Broker API (v2.11)
```

Comment:
In master, should we also make a note that the spec may contain changes which are not yet release?